mrtt-ui/src/components/EcologicalStatusAndOutcomes/EcologicalStatusAndOutcomesForm.js

```diff
@@ -31,14 +31,15 @@ import useInitializeQuestionMappedForm from '../../library/useInitializeQuestion
 import CheckboxGroupWithLabelAndController from '../CheckboxGroupWithLabelAndController'
 import { multiselectWithOtherValidationNoMinimum } from '../../validation/multiSelectWithOther'
 import { findDataItem } from '../../library/findDataItem'
+import MONITORING_FORM_TYPES from '../../constants/monitoringFormTypes'
 
 const getBiophysicalInterventions = (registrationAnswersFromServer) =>
   findDataItem(registrationAnswersFromServer, '6.2') ?? []
 
 const EcologicalStatusAndOutcomesForm = () => {
   const { site_name } = useSiteInfo()
   const validationSchema = yup.object({
-    monitoringStartDate: yup.string().nullable(),
+    monitoringStartDate: yup.string().nullable().required(language.form.required),
```

Comment:
made dates required because they are used as an identifier for the form in the UI list